components/library/flatbuffers/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Friedrich Kink
+#
+
+BUILD_STYLE=		cmake
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=         flatbuffers
+COMPONENT_VERSION=      24.3.25
+COMPONENT_SUMMARY=      FlatBuffers: Memory Efficient Serialization Library 
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      v$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:4157c5cacdb59737c5d627e47ac26b140e9ee28b1102f812b36068aab728c1ed
```

Comment:
Please give the archive a proper name. With only a version number we cannot identify it easily and name clashes have to be expected in the future.